### ISSUE TITLE
Chat -> Messaging -> New messages marker / Mark as unread

### ISSUE THREAD

**Issue by benjthayer** [status-im/status-desktop#7488]
**Summary**
Designs have been completed to replicate the 'New messages' marker and jump to most recent message in Discord. 
_Passing designs for the following to development for implementation_

**New features**
1) New messages marker with date indicating to user what messages have been posted since their last visit
2) ~~Enhancement to message context menu allowing ability to manually 'mark as unread'~~ (moved to: https://github.com/status-im/status-desktop/issues/10329)
3) 3x floating 'jump to' buttons bottom right of UI which present in specific conditions to allow the user to...
i) Jump to the start of the new messages (when the user has scrolled above the new messages marker)
ii) Jump to the latest message posted
iii) Jump to missed @mentions one by one

Sample screenshots attached

**Figma file:**
https://www.figma.com/file/Mr3rqxxgKJ2zMQ06UAKiWL/%F0%9F%92%AC-Chat%E2%8E%9CDesktop?node-id=572%3A4517

<img width="1440" alt="@mention --_ Anchor to new messages" src="https://user-images.githubusercontent.com/110033914/191574770-659b700e-37e7-426f-8dc4-e792e1f8090f.png">
<img width="1440" alt="Unread messages --_ Anchor to new messages" src="https://user-images.githubusercontent.com/110033914/191574773-1822cd38-ed86-46fc-ac7b-a62104fdc364.png">
<img width="1440" alt="Context menu" src="https://user-images.githubusercontent.com/110033914/191574787-a7e826db-29f4-405a-ab96-f38233741099.png">

**Tasks:**
- [x] #8521
- [x] #8572
- [x] #9069
- [x] #9070

**Comment by John-44** [comment]
Before starting work on this item, whoever is doing to work on it should do a briefing call with @benjthayer and myself to walk through the details of these interactions.

**Comment by benjthayer** [comment]
Hi @John-44 @iurimatias as discussed with John, this work is complete from a design perspective and ready for development. Happy to walk through anytime with dev prior to dev work commencing. Thanks

**Comment by anastasiyaig** [comment]
@jrainville @osmaczko does it still fit the goal for 0.9.0?

**Comment by jrainville** [comment]
I moved to 0.10 to finish the last two tasks in the list.
@osmaczko can you add those issues and link them in the description above. We can work on those later

**Comment by jrainville** [comment]
Moving to 0.11 since it's not a priority for 0.10
